lib/user-methods.js

```diff
@@ -62,6 +62,19 @@ function checkObjectTypes(obj, types) {
     }
 }
 
+function normalizeSyncUrl(authUrl, syncUrl) {
+    let parsedAuthUrl = new URL(authUrl);
+    let secure = (parsedAuthUrl.protocol === 'https:') ? 's' : '';
+    // Inherit ports from the Auth url
+    let port = (parsedAuthUrl.port === undefined) ? "" : `:${parsedAuthUrl.port}`;
```

Comment:
You can safely check the port for falsy-ness and save a few characters:

```suggestion
    let port = parsedAuthUrl.port ? `:${parsedAuthUrl.port}` : "";
```